frontend/src/pages/project/projectView/ProjectView.tsx

```diff
@@ -293,12 +279,15 @@ export default function ProjectView() {
                     }, 4000);
                   }}
                 >
-                  <Box sx={{ border: 1, p: 1, bgcolor: 'background.paper' }}>
-                    <Typography>Are you sure you want to delete this project</Typography>
-                    <Box display="flex"
+                  <Box sx={{ border: 1, p: 1, bgcolor: "background.paper" }}>
+                    <Typography>
+                      Are you sure you want to delete this project

```

Comment:
Maybe add quick translation to this since you're working on these files already :smile:

Reply:
No

Reply:
[a6f6947](https://github.com/SELab-2/UGent-3/pull/418/commits/a6f6947b566fa78afbdc11722c3012421a00a497)